api/testutils/kusttest/harnessenhanced.go

```diff
@@ -189,12 +189,6 @@ func (th *HarnessEnhanced) RunTransformerAndCheckResult(
 	th.AssertActualEqualsExpectedNoIdAnnotations(resMap, expected)
 }
 
-func (th *HarnessEnhanced) ErrorFromLoadAndRunTransformer(
```

Comment:
It was difficult to understand this file because there were so many helper methods. I removed this one to reduce clutter since it seemed particularly redundant.